book.toml

```diff
@@ -0,0 +1,6 @@
+[book]
+authors = ["The Matrix.org Foundation C.I.C"]
```

Comment:
Is it the Foundation?
I thought it should be Element while we're working on this as employees of Element.

Reply:
When we're working on matrix.org projects, it's under the foundation hat.